packages/calendar/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@lion/calendar",
+  "version": "0.0.0",
+  "description": "Reusable calendar component",
+  "author": "ing-bank",
+  "homepage": "https://github.com/ing-bank/lion/",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ing-bank/lion.git",
+    "directory": "packages/calendar"
+  },
+  "scripts": {
+    "prepublishOnly": "../../scripts/insert-header.js"
```

```suggestion
    "prepublishOnly": "../../scripts/npm-prepublish.js"
```

prepublish script changed

---

thanks!